yogstation/code/modules/food_and_drinks/recipes/tablecraft/recipies_spaghetti.dm

```diff
@@ -0,0 +1,9 @@
+/datum/crafting_recipe/food/lasagna
+	name = "Lasagna"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/spaghetti/boiledspaghetti = 2,
```

Comment:
Since when does lasagna take spaghetti to make

Reply:
It's this or I made it need tomato pasta (SPAGHETTI and tomato to make that). I chose to just use the ingredients instead.

Reply:
I feel like 1 spaghetti would be better resource-wise.